blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/MatrixProjectImpl.java

```diff
@@ -0,0 +1,101 @@
+package io.jenkins.blueocean.rest.impl.pipeline;
+
+/**
+ * @author Vivek Pandey
+ */
+
+import hudson.Extension;
+import hudson.matrix.MatrixProject;
```

Comment:
Neglected to add the corresponding dependency to `blueocean-pipeline-api-impl/pom.xml`, implicitly relying on `htmlpublisher` to provide it (no longer the case after https://github.com/jenkinsci/htmlpublisher-plugin/pull/287).